edb/pgsql/metaschema.py

```diff
@@ -5419,7 +5559,7 @@ def _generate_sql_information_schema() -> List[dbops.Command]:
             tableoid, xmin, cmin, xmax, cmax, ctid
         FROM pg_namespace
         WHERE nspname IN ('pg_catalog', 'pg_toast', 'information_schema',
-                          'edgedb')
+                          'edgedb', 'edgedbstd')
```

Comment:
Why do we need to expose "edgedb" and "edgedbstd" namespaces? Shouldn't everything that was in them get exposed somewhere else?

Reply:
`edgedbstd` and `edgedb` have a bunch of types that are slightly different from PG built-in types. Since they are various EdgeDB built-in types, they get used in schemas all the time and they need their definitions.